metadata.json

```diff
@@ -187,11 +187,11 @@
       "versions": [
         {
           "id": "current-community",
-          "name": "2.1.13.RELEASE (Community)"
+          "name": "2.2.6.RELEASE (Community)"
         },
         {
           "id": "current-redhat",
-          "name": "2.1.13.RELEASE (RHOAR)"
+          "name": "2.2.6.RELEASE (RHOAR)"
```

Comment:
I thought that we were supposed to change the supported release name? @cmoulliard @zekaguilar

Reply:
Good point